governing-documents/MAINTAINERS-guidelines.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: default
 title: MAINTAINERS Guidelines
-parent: Guidelines
+parent: Governing Documents
 grand_parent: LF Decentralized Trust TAC
 nav_order: 2
 ---
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
 ## Introduction
 
-All Hyperledger repositories **MUST** have a `MAINTAINERS.md` file at the top-level directory of the source code. The [SAMPLE-MAINTAINERS.md](SAMPLE-MAINTAINERS.md) can be used as a template by projects creating a new repository.
+All Linux Foundation Decentralized Trust (LFDT) repositories **MUST** have a `MAINTAINERS.md` file at the top-level directory of the source code. The [SAMPLE-MAINTAINERS.md](SAMPLE-MAINTAINERS.md) can be used as a template by projects creating a new repository.
 
-The Hyperledger Foundation GitHub organization administrators **SHOULD** periodically send out notifications about missing `MAINTAINERS.md` files.
+The LFDT GitHub organization administrators (i.e., LFDT TAC) **SHOULD** periodically send out notifications about missing `MAINTAINERS.md` files.
```

Comment:
_Question_: should this be the LFDT TAC or the project's TSC? I can see this being a concern for both bodies.

Reply:
@VRamakrishna I'd be fine with saying both. It's probably the TSC's responsibility to add the maintainers.md file (if we assume that TSC is mostly the same set of people as the project maintainers) but in the cases when it isn't it makes sense for the TSC to have this responsibility in addition to the TAC also paying attention to it.

Reply:
How about — the project TSC **MUST** monitor that all repositories have a maintainers.md file, and the TAC **SHOULD** send out periodioc notifications about missing files.

Reply:
I'm agnostic on this myself. I can say "TAC or TSC".

Reply:
Accepting Stephen's suggestion.